django/api/services/calculate_rebate.py

```diff
@@ -0,0 +1,70 @@
+from ..models.go_electric_rebate_application import GoElectricRebateApplication
+from ..models.household_member import HouseholdMember
+from ..settings import INCOME_REBATES
+
+
+def calculate_rebate_amount(cra_response, application_id):
```

Comment:
cra response is an array with multiple application ids within it potentially. Passing an `application_id` shouldn't be done here.